src/Resources/contao/templates/modules/mod_login.html5

```diff
@@ -13,9 +13,8 @@
       <?php endif; ?>
       <input type="hidden" name="FORM_SUBMIT" value="<?= $this->formId ?>">
       <input type="hidden" name="REQUEST_TOKEN" value="{{request_token}}">
-      <?php if ($this->targetPath): ?>
-        <input type="hidden" name="<?= $this->targetName ?>" value="<?= StringUtil::specialchars($this->targetPath) ?>">
-      <?php endif; ?>
+      <input type="hidden" name="redirect" value="<?= StringUtil::specialchars($this->redirect) ?>">
+      <input type="hidden" name="redirectBack" value="<?= (int) $this->redirectBack ?>">
```

Comment:
Why do we need a second parameters? We could leave the `redirect` field empty if `!$this->redirectBack`.

Reply:
Unfortunately we cannot. The process is rather complex:

 1. if *redirect back* is enabled and a URL is present, pass that URL and always use it in the success handler
 2. if *redirect back* is **not enabled** but a forward page is set, pass that URL
 3. the SuccessHandler redirects according to 1. if `redirectBack` is set. If not, it first checks for user groups and redirects to them, otherwise falls back to `redirect`